class-wxr-importer.php

```diff
@@ -1754,6 +1761,47 @@ protected function process_term( $data, $meta ) {
 	}
 
 	/**
+	 * Process and import term meta items.
+	 *
+	 * @param array $meta List of meta data arrays
+	 * @param int $term_id Term ID to associate with
+	 * @param array $term Term data
+	 * @return int|bool Number of meta items imported on success, false otherwise.
+	 */
+	protected function process_term_meta( $meta, $term_id, $term ) {
+		if ( empty( $meta ) ) {
+			return true;
+		}
+
+		foreach ( $meta as $meta_item ) {
+			/**
+			 * Pre-process term meta data.
+			 *
+			 * @param array $meta_item Meta data. (Return empty to skip.)
+			 * @param int $term_id Term the meta is attached to.
+			 */
+			$meta_item = apply_filters( 'wxr_importer.pre_process.term_meta', $meta_item, $term_id );
```

Comment:
Continuing from https://github.com/humanmade/WordPress-Importer/pull/18#discussion_r105027093

I think the term_id is different to the term data, so we can't just pass term.